### ISSUE TITLE
Should I go with privacore?

### ISSUE THREAD

**Issue by unisqu** [gigablast/open-source-search-engine#123]
Should I go with privacore? 1.7 mil indexed within around 2 days, disks already using 95GB for 1.7 mil items. for 17 mil ~ 950GB? 170mil items 9.5TB? this is a bit too much for this?

**Comment by unisqu** [comment]
Ok I just converted to privacore. It does feel less heavy on disk but the load havent really subsided much. from 50 to 30-40... so it's actually quite fine.

**Comment by dp40oz** [comment]
Yea these claims here are way off. I've never come close to 28.6gb per 1 million documents. Usually more like 50-55gb. So about double. 

> 1 million web pages requires 28.6GB of drive space. That includes the index, meta information and the compressed HTML of all the web pages. That is 28.6K of disk per HTML web page.

**Comment by unisqu** [comment]
i think it's because I did the duplicate search thing. my max doc size setting is 100kb

**Comment by lisawebcoder** [comment]
hello i been trying to build install and run the software from source or the other methos but always fail 
gb gile not found
also i made a wiki api app but i cant seem to call the Gb api endpoints
do you have the correct steps or scripts to run the full code or call the api in a webpage gui searchbox?

well thxz 
Lisa

**Comment by onlyjob** [comment]
According to _primacore_ themselves, you shouldn't. Read "A word of warning" paragraph from their statement: https://web.archive.org/web/20190921180535/https://privacore.github.io/

> **A word of warning**
> The search engine was based on the Gigablast open-source search engine by Matt Wells. Normally I endorse the “If you don’t have something nice to say, don’t say anything at all” saying, but… That was one of the biggest mistakes we made. Looking back, that codebase was outdated and we ended up putting way too much time and money into bug-fixing and optimizing it instead of adding new features. It was not as flexible or feature rich as it needed to be, to be the foundation of a modern search engine. We came close on a number of occasions to making a deal with Matt to develop it further, but every time he ended up disappearing. We will leave our version available for the foreseeable future, but our recommendation is clear – DO NOT USE IT – not ours, and especially not the original. Start somewhere else if you want to build a new search engine. Do not base your project on a one-man project, no matter how confidently it is “sold” to you.

[Primacore fork](https://github.com/privacore/open-source-search-engine) is now archived. The have managed to make some good improvements to the Gigablast code but got burned by the effort apparently not realising the extent of commitment required to maintain and develop Gigablast.

IMHO we need a community effort _here_ to backport fixes from forks and to carry on the development.

**Comment by gigablast** [comment]
If only there was a source of some serious funding, things would be a lot different.

**Comment by onlyjob** [comment]
> If only there was a source of some serious funding, things would be a lot different.

Indeed. Or community interest to maintain an independent crawler, for competition or other numerous reasons.

Was any crowdfunding or fundraising attempted?

By the way thank you sincerely, @gigablast, for your efforts.

**Comment by gigablast** [comment]
no, i haven't attempted anything like that. but to make a serious effort at this would be in the millions of dollars. not like $50k that is typical on those platforms.
